### PR TITLE
Update emojis.js

### PR DESCRIPTION
**Pull request by mgscuteri**
Added extra word associations.  (This time without any trailing spaces so CI should succeed!)

### CODE REVIEW

File: src/utils/emojis.js [hunk 1]
```diff
@@ -80,7 +79,15 @@ const EMOJI_MAP = new Map([
   ['halloween', ':ghost:'],
   ['fire', ':fire:'],
   ['hot', ':fire:'],
+  ['heat', ':fire:'],
   ['burn', ':fire:'],
+  ['prison', ':gun:'],
+  ['Lucky', ':raised_hands:'],
```
Comment (micthiesen) [inline src/utils/emojis.js:85]: Mind making this lowercase? The REGEX is case insensitive so functionally it doesn't matter. It's just nice to be consistent.

**Review by micthiesen — CHANGES_REQUESTED** [review]
I'm not sure why CI is failing; everything looks good to me. I'll see if I can figure out what's going on.

And thanks for the changes! This emoji list needs some attention.

File: src/utils/emojis.js [hunk 2]
```diff
@@ -80,7 +79,15 @@ const EMOJI_MAP = new Map([
...
+  ['electric', ':zap:'],
+  ['thunder', ':thunder_cloud_and_rain:'],
+  ['storm', ':thunder_cloud_and_rain:'],
+  ['money', ':money_mouth_face:'],
+  ['dollar', ':money_mouth_face:'],
```
Comment (micthiesen) [inline src/utils/emojis.js:90]: There are 2 extra spaces at the end of this line, which is why CI is failing 🙂

File: src/utils/emojis.js (hunk 1 above)
Comment (mgscuteri) [reply src/utils/emojis.js:85]: Sure, I made a separate pull request.  How do you feel about adding non standard emoji's  (ie party_parrot)

Will they fall back to a default?

Comment (micthiesen) [reply src/utils/emojis.js:85]: Unfortunately it won't fall back to our defaults (the music notes / headphones) so I think we should stick with standard emojis

**Pull request closed without merging**